Source/Particles/PhysicalParticleContainer.cpp

```diff
@@ -986,6 +986,29 @@ PhysicalParticleContainer::AddPlasma (int lev, RealBox part_realbox)
                     r = 1;
                 }
                 pcounts[index] = num_ppc*r;
+                // update pcount by checking which particles have non-zero density
+                int flag_pcount = 0;
+                for (int ip = 0; ip < num_ppc*r; ++ip) {
+                    const XDim3 r_loc =
+                        inj_pos->getPositionUnitBox(ip, lrrfac, amrex::RandomEngine{});
+                    amrex::Real dens1 = inj_rho->getDensity(lo.x, lo.y, lo.z);
```

Comment:
Are you sure this will do the correct thing in 1D / 2D / 2D RZ geometry?

Reply:
oops. forgot to remove this when cleaning it up

Reply:
and will fix for 1D/2D/RZ